includes/ucf-resource-link-posttype.php

```diff
@@ -64,24 +64,74 @@ public static function register_metafields( $post ) {
 			wp_nonce_field( 'ucf_resource_link_nonce_save', 'ucf_resource_link_nonce' );
 			$url = get_post_meta( $post->ID, 'ucf_resource_link_url', TRUE );
 			$admins = get_post_meta( $post->ID, 'ucf_resource_link_admins', TRUE );
+			$facebook = get_post_meta( $post->ID, 'ucf_resource_facebook_url', TRUE );
+			$twitter = get_post_meta( $post->ID, 'ucf_resource_twitter_url', TRUE );
+			$instagram = get_post_meta( $post->ID, 'ucf_resource_instagram_url', TRUE );
+			$linkedin = get_post_meta( $post->ID, 'ucf_resource_linkedin_url', TRUE );
+			$youtube = get_post_meta( $post->ID, 'ucf_resource_youtube_url', TRUE );
 ?>
 			<table class="form-table">
 				<tbody>
 					<tr>
 						<th>
-							<label class="block" for="ucf_resource_link_url"><strong>URL</strong></label>
+							<label class="block" for="ucf_resource_link_url"><strong>Website URL</strong></label>
 						</th>
 						<td>
 							<input type="text" id="ucf_resource_link_url" name="ucf_resource_link_url" class="regular-text" <?php echo ( ! empty( $url ) ) ? 'value="' . $url . '"' : ''; ?>>
 						</td>
 					</tr>
+					<tr>
+						<th>
+							<label class="block" for="ucf_resource_facebook_url"><strong>Facebook URL</strong></label>
+						</th>
+						<td>
+							<input type="text" id="ucf_resource_facebook_url" name="ucf_resource_facebook_url" class="regular-text" <?php echo ( ! empty( $facebook ) ) ? 'value="' . $facebook . '"' : ''; ?>>
```

Comment:
These could be simplified, since an empty/null value on a text input is fine.

```
<input type="text" id="ucf_resource_facebook_url" name="ucf_resource_facebook_url" class="regular-text" value="<?php echo $facebook; ?>">
```